### PR TITLE
Code to check whether a LinkedList contains Palindromic data or not

### LINKED ISSUE #2067: C++ Program to check whether a singly linked List is a palindrome or not
Can I get assigned to writing the code of the problem mentioned in the title of this issue?

### PR DESCRIPTION
**Pull request by disha2sinha**
Fixes #2067 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Check_Palindrome_for_LinkedList/Palindrome.cpp [hunk 1]
```diff
@@ -0,0 +1,138 @@
+#include <bits/stdc++.h>
```
Comment (jainaman224) [inline Check_Palindrome_for_LinkedList/Palindrome.cpp]: Fix file name.

File: Check_Palindrome_for_LinkedList/Palindrome.cpp [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+using namespace std;
+
+class Node
+{
+public:
+    string data;
+    Node *next;
+};
```
Comment (jainaman224) [inline Check_Palindrome_for_LinkedList/Palindrome.cpp]: Add new lines after the function end.

File: Check_Palindrome_for_LinkedList/Palindrome.cpp [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+dca
+ENTER 1 TO CONTINUE INSERTING AND 0 TO STOP:
+1
+ENTER ITEM:
+1
+ENTER 1 TO CONTINUE INSERTING AND 0 TO STOP:
+0
+ YOUR LINKED LIST LOOKS LIKE :
+ |_START |_1_|_|-> |_ab_|_|-> |_dca_|_|-> |_1_|_|->[NULL]
+ Not Palindrome
```
Comment (jainaman224) [inline Check_Palindrome_for_LinkedList/Palindrome.cpp]: Please close comment tag.

File: Check_Palindrome_for_LinkedList/Palindrome.cpp [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+{
+public:
+    string data;
+    Node *next;
+};
+Node *start;
+/*Function to insert values to the linked_list at the end*/
+void insert_end()
+{
+    Node *p = start;
```
Comment (jainaman224) [inline Check_Palindrome_for_LinkedList/Palindrome.cpp]: Add new lines around the blocks to make it more readable.

**Comment by disha2sinha** [comment]
Changes are made.

**Pull request merged**